goenv/version.go

```diff
@@ -54,7 +59,18 @@ func GetGorootVersion() (major, minor int, err error) {
 	if err != nil {
 		return 0, 0, fmt.Errorf("failed to parse version: %s", err)
 	}
-	return
+
+	return major, minor, nil
+}
+
+// WantGoVersion returns true if Go version s is >= major and minor.
+// Returns false if s is not a valid Go version string. See [ParseGoVersion] for more information.
+func WantGoVersion(s string, major, minor int) bool {
+	ma, mi, err := ParseGoVersion(s)
+	if err != nil {
+		return false
+	}
+	return ma >= major && mi >= minor
```

Comment:
What if we're on Go 2.2 and the code is written for Go 1.20? It would fail this check, though 2.20 would work.

```suggestion
	return ma > major || (ma == major && mi >= minor)
```

All very hypothetical of course, since I don't think there will ever be a Go 2.0 (and the compatibility situation may not be what we expect when that happens). So perhaps we can just ignore the possibility of Go 2.0 for now.